maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/StatelessXmlReporter.java

```diff
@@ -292,15 +293,15 @@ private FileOutputStream getOutputStream( WrappedReportEntry testSetReportEntry
 
         try
         {
-            return new FileOutputStream( reportFile );
+            return new BufferedOutputStream( new FileOutputStream( reportFile ), 16 * 1024 );
```

Comment:
Is this relevant for this changeset? I agree the reporter should use either directly the stream or the writer. However I see this as a separate problem. 

I can have a look at it as well, but it appears rather tricky and I would like to have it isolated from this very change. See this part: https://github.com/jerrinot/maven-surefire/blob/b66b524a42d6cbc2b23225d72b67671d25d3fc05/maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/StatelessXmlReporter.java#L442-L442